apps/frontend/app/(client)/(main)/course/_components/RegisterCourseButton.tsx

```diff
@@ -0,0 +1,168 @@
+'use client'
+
+import { Button } from '@/components/shadcn/button'
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle
+} from '@/components/shadcn/dialog'
+import { Input } from '@/components/shadcn/input'
+import { Separator } from '@/components/shadcn/separator'
+import { isHttpError, safeFetcherWithAuth } from '@/libs/utils'
+import plusCircleIcon from '@/public/icons/plus-circle.svg'
+import type { Course } from '@/types/type'
+import { useQueryClient } from '@tanstack/react-query'
+import Image from 'next/image'
+import { useEffect, useState } from 'react'
+import { toast } from 'sonner'
+
+export function RegisterCourseButton() {
+  const [isDialogOpen, setIsDialogOpen] = useState(false)
+  return (
+    <>
+      <Button variant="slate" onClick={() => setIsDialogOpen(true)}>
+        <div className="border-primary flex h-[32px] w-[127px] items-center justify-center gap-2 rounded-full border">
```

Comment:
사소한건데 h-8로 바꿔도 좋을 것 같아요!
```suggestion
        <div className="border-primary flex h-8 w-[127px] items-center justify-center gap-2 rounded-full border">
```